app/assets/javascripts/discourse/app/controllers/user.js

```diff
@@ -250,7 +255,7 @@ export default Controller.extend(CanCheckEmails, {
     },
 
     updateNotificationLevel(level) {
-      return this.model.updateNotificationLevel({ level });
+      return this.model.updateNotificationLevel(level);
```

Comment:
This should still be `({ level })` I think

Reply:
See 

https://github.com/discourse/discourse/blob/5423d4644201c94129d3590375bb5aed44681141/app/assets/javascripts/discourse/app/models/user.js#L783-L786

Reply:
@martin-brennan But... the ability to mute/ignore a user is _broken_ here. It literally fixes the ability to do so.
`return this.model.updateNotificationLevel({ level });` was the issue, because `level` is already the object. It's passing in `{ level: { level: "mute" } }` for example. Changing this to `return this.model.updateNotificationLevel(level);` fixes that issue.

> It took some digging and a trip to the meta topic to realize that this is fixing a bug where ignored is mistakenly used instead of ignore

This isn't the only problem. Muting and ignoring is broken right now because of what I explained above

Reply:
Also the prefix `FIX:` is explaining that it is fixing the ability, not adding a new feature!

Reply:
The fix looks good, but I see where Martin is coming from, the parameter `level` in `updateNotificationLevel(level)` is confusing, because it is an object with a `level: "smth"` property. Maybe we rename the param, as in `updateNotificationLevel(params)`, and pass to the model `return this.model.updateNotificationLevel(params);`. 

Minor rename, but should be cleaner for the next person.

Reply:
Ah I see what you are saying with `level` now, my bad. However I think one of my points point still stands. The commit message `FIX: Bug setting notification level to muted/ignored on user page` is definitely an improvement, however there is no PR description here to describe the problem and the fix, and now there is no detail on the commit message either.

We should be striving to include a good amount of context in commit messages for reviewers, people using our open source code, and developers looking back on this in future with `git blame` :) Things like linking to the commit that introduce the bug (my one at https://github.com/discourse/discourse/commit/ca93e5e68b5eb6843b112d72ba7f108004d4c674) leaves a helpful paper trail too :)

Thank you for fixing the bug I introduced!